pom.xml

```diff
@@ -14,11 +14,6 @@
     </properties>
     <dependencies>
         <dependency>
-            <groupId>com.okta.libraries.framework</groupId>
-            <artifactId>aperture-reporter</artifactId>
-            <version>${aperture-reporter.version}</version>
```

Comment:
The corresponding line, <aperture-reporter.version>1.0.0-000079.000042.e302966</aperture-reporter.version> could be removed.

You could also remove.
```
    <profiles>
        <profile>
            <id>ci-run</id>
            <properties>
                <test.listener>listener</test.listener>
            </properties>
        </profile>
    </profiles>
```
And then the build_and_test.sh script will have to see some changes, similar to how it is done here:
https://github.com/okta/org-creator/blob/master/scripts/build_and_test.sh